src/pages/Login/Login.js

```diff
@@ -1,7 +1,100 @@
+import { useState } from "react"
+import { useAuth } from "../../Context/AuthContext/AuthContext"
+import { Link } from "react-router-dom";
 export default function Login(){
+
+    const {handleLoginData} = useAuth();
+
+    const [login, setLogin] = useState({
+        email:'',
+        password:''
+    }) 
+    const [isError, setIsError] = useState('')
+    const handleData = (e)=>{
+        setLogin({...login, [e.target.name]:e.target.value})
+    }
+    console.log(login)
+
+
+    const loginHandler = (e) =>{
+        const {email, password} = login
+        e.preventDefault();
+        if(email!== '' && password!==''){
+            handleLoginData(email,password)
+        }
+        else{
+            setIsError('Please fill all the field')
+        }
+        setIsError('')
+    }
+    const test =  {
+       email:"adarshbalika@gmail.com",
+        password:"adarshbalika"
+    }
+
     return(
-        <div>
-            <h3>This is Login page</h3>
+        <div class="flex-align-center m-5">
+        <div class="form box-shadow pd-2 ">
+            <h2 class="mt-1 mb-2 flex-align-center">Login</h2>
+            <form class="form" onSubmit={loginHandler}>
+                <div class="input-with-icons">
+                    <i class="bi bi-person-fill input-icon"></i>
+                    <input 
+                        type="email" 
+                        placeholder="Enter your Email"
+                        class="icon-input"
+                        name="email"
+                        value={login.email}
+                        onChange={handleData}
+                    />
+                </div>
+                <div class="input-with-icons">
+                    <i class="fa fa-lock input-icon"></i>
+                    <input type="password"
+                        placeholder="Enter your Password"
+                        class="icon-input"
+                        name="password"
+                        value={login.password}
+                        onChange={handleData}
+                    />
+                </div>
+               {isError == '' ? <></> :
+               <div className="alert alert-danger h6">{isError}</div>
+                } 
+            <div class="gap-2 flex-align-center">
+                <div class="h6">
+                    <input 
+                    type="checkbox" 
+                    id="checkbox"
+                    required
+                    />
+                    
+                    <span class="pd--5">
+                        Remember me</span></div>
+                <a href="#"><button class="btn-link-primary h6">Forgot your password?</button></a>
```

Comment:
I think you forgot to remove anchor tag. If you put it on purposely then I would recommend you to use Link component provided by react-router dom. Other than that your codes look clean and readable. Great work.

Reply:
ya I just forgot to remove it thank you for review